iib/workers/config.py

```diff
@@ -90,6 +90,9 @@ class Config(object):
     # path where catalog resides in fbc_fragment
     # might need to be changed, currently based on test fbc-fragment
     fbc_fragment_catalog_path: str = '/configs'
+    # path where operator deprecations will be stored
+    # it's a sub-directory of fbc_fragment_catalog_path
+    operator_deprecations_dir: str = '_operator-deprecations-content'
```

Comment:
Purposefully not adding it to README so we don't advertise the change before it's ready. Will be added to README in #713

Reply:
Actually I did not add in the README of the other PR as well as we haven't added any of the dirs or paths in readme like fbc_fragment_catalog_path or hidden_index_db_path

Reply:
ack, we probably should add them.